<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QwertyKb</class>
 <widget class="QWidget" name="QwertyKb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pbQ">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbW">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>52</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>w</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbE">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>10</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbR">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>10</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>r</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbT">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>10</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>t</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbY">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>10</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbU">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>10</y>
     <width>52</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbI">
   <property name="geometry">
    <rect>
     <x>422</x>
     <y>10</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbO">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>10</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbP">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>51</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>p</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbG">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>77</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>g</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbL">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>77</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbK">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>77</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>k</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbJ">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>77</y>
     <width>52</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>j</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbS">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>77</y>
     <width>52</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbF">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>77</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>f</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbH">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>77</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbD">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>77</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>d</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbA">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>77</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbV">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>143</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBack">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>143</y>
     <width>111</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbM">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>143</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbN">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>143</y>
     <width>52</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbZ">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>143</y>
     <width>52</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbC">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>143</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbB">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>143</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>b</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbX">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>143</y>
     <width>53</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEnter">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>141</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDot">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>143</y>
     <width>51</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbComma">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>71</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNum">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>210</y>
     <width>61</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>123</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSpace">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>281</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>space</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEnter_2">
   <property name="geometry">
    <rect>
     <x>461</x>
     <y>210</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb3">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>10</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb9">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>143</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb2">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>10</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb8">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>143</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>77</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb5">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>77</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLetter">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>210</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>abcd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSpace_2">
   <property name="geometry">
    <rect>
     <x>461</x>
     <y>77</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>space</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb7">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>143</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBack_2">
   <property name="geometry">
    <rect>
     <x>461</x>
     <y>10</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDash">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>210</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb0">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>210</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb6">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>77</y>
     <width>135</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
